web/models.py

```
@@ -2804,3 +2804,193 @@ class ScheduledPost(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    
+    class Meta:
+        ordering = ['-is_featured', 'name']
+        verbose_name = "Chapter"
+        verbose_name_plural = "Chapters"
+    
+    def __str__(self):
+        return f"{self.name} ({self.region})"
+    
+    def save(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `save`

Add return type annotation: `None`

(ANN201)